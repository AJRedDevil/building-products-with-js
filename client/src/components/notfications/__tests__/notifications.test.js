// npm packages
import React from 'react';
import configureMockStore from 'redux-mock-store';

// our packages
import Notifications from '../index';

// create mockstore
const mockStore = configureMockStore();

test('# Notifications', () => {
  const store = mockStore({});
  const wrapper = shallow(<Notifications store={store} />);
  expect(wrapper).toMatchSnapshot();
});
