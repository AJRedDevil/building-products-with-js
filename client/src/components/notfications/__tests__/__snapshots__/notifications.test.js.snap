// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Notifications 1`] = `
<Notifications
  dispatch={[Function]}
  notifications={
    Array [
      Object {
        "alertType": "",
        "id": 0,
        "text": "",
      },
    ]
  }
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
  storeSubscription={
    Subscription {
      "listeners": Object {
        "clear": [Function],
        "get": [Function],
        "notify": [Function],
        "subscribe": [Function],
      },
      "onStateChange": [Function],
      "parentSub": undefined,
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "unsubscribe": [Function],
    }
  }
/>
`;
